demos/python_demos/3d_segmentation_demo/3d_segmentation_demo.py

```diff
@@ -248,13 +280,17 @@ def main():
     if not os.path.exists(args.path_to_input_data):
         raise AttributeError("Path to input data: '{}' does not exist".format(args.path_to_input_data))
 
-    is_nifti_data = os.path.isdir(args.path_to_input_data)
+    input_type = get_input_type(args.path_to_input_data)
+    is_nifti_data = (input_type == NIFTI_FILE or input_type == NIFTI_FOLDER)
 
-    if is_nifti_data:
+    if input_type == NIFTI_FOLDER:
         series_name = find_series_name(args.path_to_input_data)
         original_data, data_crop, affine, original_size, bbox = \
-            read_image(args.path_to_input_data, series_name=series_name, sizes=(h, w, d))
+            read_image(args.path_to_input_data, data_name=series_name, sizes=(h, w, d))
 
+    elif input_type == NIFTI_FILE:
```

Comment:
You should document this new input type in the README.

Reply:
Done